src/content/getting-started/cloud-provider-accounts/cluster-api/aws/index.md

```diff
@@ -0,0 +1,118 @@
+The following overview lists the limits that have to be adjusted in order to use
+the account to operate Giant Swarm Cluster API workload clusters.
+
+Adjusting a service limit requires a support case in the
+[AWS Support Center](https://console.aws.amazon.com/support/home),
+where a specific entry form is provided for this type of case. Each limit type
+requires a separate case. When creating these, make sure to be logged in to the
+AWS account you want to adjust the limits for, and always select the correct
+region.
+
+The screenshot below shows the entry form.
+
+![Screenshot](aws-service-limits.png)
+
+These are the limit increases to be requested, grouped by limit type:
+- All AWS accounts
+    - VPC
+        - Routes per route table: **200**
+        - VPCs per region: **50**
+        - IPv4 CIDR blocks per VPC: **50**
+    - Elastic Load Balancers
+        - Application and Classic Load Balancers per region: **100**
+    - Auto Scaling
+        - Auto Scaling Groups per region: **250**
+        - Launch Configurations per region: **500**
+    - EC2 Instances
+        - m5.xlarge per region: **250**
+        - m5.2xlarge per region: **250**
+        - other instance types to be used as workers: increase accordingly
+    - EC2 Spot Instances
+        - For every primary instance type you tend to use spot instances with, set the limit according to your needs.
+- AWS account intended to run public clusters (outgoing traffic goes to public internet)
+    - VPC
+        - NAT Gateway per Availability Zone per region: **50**
+    - Elastic IP
+        - New VPC Elastic IP Address Limit per region: **50**
+
+- AWS account intended to run private clusters (outgoing traffic goes to public internet only thru proxy)
+    - Route 53 Resolver
+        - Endpoints per AWS Region: **100**
+
+(Please extend the list of EC2 instances to also contain the types you need frequently.)
+
+## Configure the `AWSClusterRoleIdentity`
+When you want to create a new Cluster API workload cluster in a new AWS account you need to create a CR `AWSClusterRoleIdentity` which will reference the role in the new AWS Account.
+
+```yaml
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AWSClusterRoleIdentity
+metadata:
+  labels:
+    cluster.x-k8s.io/watch-filter: capi
+  name: ${ROLE_NAME}
+spec:
+  allowedNamespaces:
+    list: null
+    selector: {}
+  roleARN: ${ROLE_ARN}
+  sourceIdentityRef:
+    kind: AWSClusterControllerIdentity
+    name: default
+```
+Where:
+* `${ROLE_NAME}` is a short unique name referencing AWS account (ie: `development`, `sandbox` or `staging2`)
+* `${ROLE_ARN}` is full ARN of the role created in previous step with `giantswarm-aws-account-prerequisites` repository.
+
+
+This CR needs to be created only once for each AWS Account. It can be then refered in the `cluster-aws` repo in the value [`aws.awsClusterRoleIdentityName`](https://github.com/giantswarm/cluster-aws/blob/master/helm/cluster-aws/values.yaml#L14).
```

Comment:
```suggestion
This CR needs to be created only once for each AWS Account. It can be then referenced in the `cluster-aws` repo in the value [`aws.awsClusterRoleIdentityName`](https://github.com/giantswarm/cluster-aws/blob/master/helm/cluster-aws/values.yaml#L14).
```